Project.toml

```diff
@@ -8,7 +8,6 @@ FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
-FileIO = "1.4"
```

Comment:
Please specify a compat entry for FileIO as well, the dev version you have locally should be greater than 1.4 so no issues with leaving the compat entry here. Makes sense?